scm/test_helper.go

```diff
@@ -18,11 +17,7 @@ func MockSCMManager(t *testing.T) (SCM, *Manager) {
 		"test",
 		&app.Config{},
 	}
-	sc := NewFakeSCMClient()
```

Comment:
We should avoid using non-descriptive names such as `test` for arguments/fields because it can difficult to know which field cause a problem.